CHANGES.md

```diff
@@ -7,6 +7,7 @@
   with observe_dask_progress('Writing dataset', 100):
       dataset.to_zarr(store)  
   ```
+* Avoid accessing variable data during normalisation. (#392)
```

Comment:
Please tell the user what you mean by "normalisation" and why the change was made.